ruby/parser.js

```diff
@@ -162,7 +184,7 @@
     } else if (typeof value === 'undefined' || value === null) {
         stringified = 'nil'
     } else if (typeof value === 'string') {
-        stringified = `'${value}'`
+        stringified = `'${value.replace(/'/ig, `\\'`)}'`
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
        stringified = `'${value.replace(/'/ig, "\\'")}'`
```